Functions/xASL_test_GetLogContent.m

```diff
@@ -0,0 +1,308 @@
+function [logContent] = xASL_test_GetLogContent(rootDir, printContent, storeRelativePath, exportTable)
+%xASL_test_GetLogContent Get warnings and errors from log files
+%
+% FORMAT: [logContent] = xASL_test_GetLogContent(rootDir, [printContent], [storeRelativePath], [exportTable])
+%
+% INPUT:
+%        rootDir            - Case root directory (OPTIONAL, DEFAULT = user input)
+%        printContent       - Print warnings and errors (OPTIONAL, DEFAULT = false, BOOLEAN)
+%        storeRelativePath  - Store relative path in logContent table instead of module name (OPTIONAL, DEFAULT = true, BOOLEAN)
+%        exportTable        - Export a tsv or xlsx file containing the log content (OPTIONAL, DEFAULT = 1, BOOLEAN)
+%                             (0 = no export, 1 = TSV export, 2 = XLSX export)
+%
+% OUTPUT:
+%        logContent         - table containing warnings and errors
+%
+% -----------------------------------------------------------------------------------------------------------------------------------------------------
+% DESCRIPTION:      Get warnings and errors from log files.
+%
+% -----------------------------------------------------------------------------------------------------------------------------------------------------
+%
+% EXAMPLE:          % Run the GetLogContent script, do not print the messages, store the full path and export an XLSX
```

Comment:
Very good to describe what actually this example does! We haven't done that in most other functions :)